docs/articles/documentation/using-testcafe/common-concepts/reporters.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: docs
+title: Reporters
+permalink: /documentation/using-testcafe/common-concepts/reporters/
+---
+# Reporters
+
+Reporters are plugins used to output test run reports in a certain format.
+
+TestCafe ships with the following reporters:
+
+* [spec](https://github.com/DevExpress/testcafe-reporter-spec) - used by default
+* [list](https://github.com/DevExpress/testcafe-reporter-list)
+* [minimal](https://github.com/DevExpress/testcafe-reporter-minimal)
+* [xUnit](https://github.com/DevExpress/testcafe-reporter-xunit)
+* [JSON](https://github.com/DevExpress/testcafe-reporter-json)
+
+You can also create a [custom repoter](/testcafe/documentation/extending-testcafe/creating-custom-reporter/) that will fit your needs.
+
+You can select a required reporter when working both through the [command line](/testcafe/documentation/using-testcafe/command-line-interface/#r-name-reporter-name) and [programmatically](/testcafe/documentation/using-testcafe/programming-interface/Runner/#reporter).
```

Comment:
We need to mention somewhere, how to use and install reporters. 
1. You need to install reporter from npm globally or locally
2. TestCafe reporters always have name `testcafe-reporter-{alias}`
3. Then you can specify them by `alias`